packages/ubuntu_wsl_setup/lib/wizard.dart

```diff
@@ -29,11 +23,26 @@ class UbuntuWslSetupWizard extends StatelessWidget {
             if ((settings.arguments as bool?) == true) {
               return Routes.advancedSetup;
             }
-            return Routes.configurationUI;
+            return Routes.setupComplete;
           default:
             return null;
         }
       },
     );
   }
 }
+
+class UbuntuWslReconfigureWizard extends StatelessWidget {
+  const UbuntuWslReconfigureWizard({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Wizard(
+      initialRoute: Routes.advancedSetup,
+      routes: <String, WidgetBuilder>{
+        Routes.advancedSetup: AdvancedSetupPage.create,
+        Routes.configurationUI: ConfigurationUIPage.create,
```

Comment:
Isn't this missing `Routes.setupComplete` in the list of routes? Otherwise, when clicking "Save" on the configuration UI page, what will happen when `Wizard.next()` is called?

Reply:
Thanks, I removed the next() call because it was there only to make all pages clickable through in the beginning.